src/measure_theory/measure_space.lean

```diff
@@ -835,6 +860,9 @@ lemma preimage_null_of_map_null {f : α → β} (hf : measurable f) {s : set β}
   (hs : map f μ s = 0) : μ (f ⁻¹' s) = 0 :=
 nonpos_iff_eq_zero.mp $ (le_map_apply hf s).trans_eq hs
 
+lemma tendsto_ae_map {f : α → β} (hf : measurable f) : tendsto f μ.ae (map f μ).ae :=
```

Comment:
By the same trick as above, isn't it true even when `f` is not measurable?

Reply:
No, if `f` is not measurable then `map f μ = 0`, `(map f μ).ae = bot`.

Reply:
Oops, wrong direction in my head. Sorry!